### ISSUE TITLE
Do an editing pass / cleanup of the vignette

### ISSUE THREAD

**Issue by mjskay** [MUCollective/pgog#83]
Some things that it needs:

* [ ] An intro to the package and a description of the basic concepts and how it works
  * [ ] Maybe some introduction to an abstract notation describing what's happening
* [ ] Prose around the various pieces introducing and explaining them
* [ ] Readable, consistent code style, especially to make it easy to see the parts of a spec. I'd suggest something like:

  ```r
  data_frame %>%
    ggplot(aes(
      var = value,
      var2 = value2
    )) +
    geom_whatever() +
    whatever_else()
  ```

  or: 

  ```r
  data_frame %>%
    ggplot() +
    geom_whatever(aes(
      var = value,
      var2 = value2
    )) +
    whatever_else()
  ```

  or: 

  ```r
  ggplot(data_frame) +
    geom_whatever(aes(
      var = value,
      var2 = value2
    )) +
    whatever_else()
  ```

Basically something that makes it easy to read the value of each aesthetic mapping and which has consistent indentation (indenting out to parens on previous lines, by contrast, creates long lines and indentation amounts that don't reflect indentation level). The advantage of the pipe forms is that if you need to insert other data transformations ahead of the ggplot call you can do so, but if no examples end up needing that it's probably not necessary.
